### PR TITLE
feature(nyp): update sqil atari config

### PR DESCRIPTION
**Pull request by Will-Nie**
## Description

1. Update the sqil configs for qbert, spaceinvader and pong. The detailed five seeds results are as follows:
Screenshot 2022-03-01 at 6 33 58 PM

Screenshot 2022-03-01 at 6 42 17 PM

Screenshot 2022-03-01 at 6 50 20 PM

2. fix the bugs (expert config must have the same ``n_sample`` collected ) to decouple sqil configs from the expert config.

## Check List

- [x] merge the latest version source branch/repo, and resolve all the conflicts
- [x] pass style check
- [x] pass all the tests

### CODE REVIEW

File: ding/entry/serial_entry_sqil.py [hunk 1]
```diff
@@ -59,6 +59,7 @@ def serial_pipeline_sqil(
     expert_cfg = compile_config(
         expert_cfg, seed=seed, env=env_fn, auto=True, create_cfg=expert_create_cfg, save_cfg=True
     )
+    expert_cfg.policy.collect.n_sample = cfg.policy.collect.n_sample
```
Comment (PaParaZz1) [inline ding/entry/serial_entry_sqil.py:62]: why add this line? add some comments

File: dizoo/atari/config/serial/pong/pong_sqil_config.py [hunk 2]
```diff
@@ -44,11 +44,11 @@
         type='atari',
         import_names=['dizoo.atari.envs.atari_env'],
     ),
-    env_manager=dict(type='subprocess'),
+    env_manager=dict(type='base', force_reproducibility=True),
```
Comment (PaParaZz1) [inline dizoo/atari/config/serial/pong/pong_sqil_config.py:47]: we have change `force_reproducibility` to `reset_inplace`, add don't add other arguments in create_cfg except `type` and `import_names`

File: dizoo/atari/config/serial/pong/pong_sqil_config.py [hunk 3]
```diff
@@ -44,11 +44,11 @@
...
     policy=dict(type='sql'),
 )
 pong_sqil_create_config = EasyDict(pong_sqil_create_config)
 create_config = pong_sqil_create_config
 
 if __name__ == '__main__':
-    serial_pipeline((main_config, create_config), seed=0)
+    serial_pipeline_sqil('pong_sqil_config.py', 'pong_dqn_config.py', seed=0)
```
Comment (PaParaZz1) [inline dizoo/atari/config/serial/pong/pong_sqil_config.py:54]: why do this modification

**Review by PaParaZz1 — CHANGES_REQUESTED** [review]

File: dizoo/atari/config/serial/pong/pong_sqil_config.py (hunk 2 above)
Comment (Will-Nie) [reply dizoo/atari/config/serial/pong/pong_sqil_config.py:47]: thx！ corrected

File: ding/entry/serial_entry_sqil.py (hunk 1 above)
Comment (Will-Nie) [reply ding/entry/serial_entry_sqil.py:62]: The purpose of adding this line is to fix the bug that if n_sample in sqil is different from that in the expert config, the code will report a bug in forward learn phase

**Review by PaParaZz1 — APPROVED** [review]

**Pull request merged**